components/rootless_studio/default/profile

```diff
@@ -40,12 +40,12 @@ alias fgrep='fgrep --color=auto'
 export TERMINFO
 TERMINFO=$(hab pkg path core/ncurses)/share/terminfo
 
-emacs() {
-  if command -v emacs > /dev/null; then
-    emacs "$@"
-  else
-    mg "$@"
-  fi
+emacs () {
+    if type -P emacs > /dev/null; then
+        "$(type -P emacs)" "$@";
+    else
+        mg "$@";
```

Comment:
Are the EOL `;` necessary?

Reply:
I was just noticing that.  I'm not sure where those came from, as my master branch doesn't have them and I didn't explicitly add them.  I'll remove them.